packages/astro-integration-angular/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import viteAngular from '@analogjs/vite-plugin-angular';
+import { AstroIntegration, AstroRenderer } from 'astro';
+
+function getRenderer(): AstroRenderer {
+  return {
+    name: '@analogjs/astro-integration-angular',
+    clientEntrypoint: '@analogjs/astro-integration-angular/client.js',
+    serverEntrypoint: '@analogjs/astro-integration-angular/server.js',
+  };
+}
+
+function getViteConfiguration() {
+  return {
+    optimizeDeps: {
+      include: ['@analogjs/astro-integration-angular/client.js'],
+      exclude: ['@analogjs/astro-integration-angular/server.js'],
+    },
+    plugins: [(viteAngular as any).default()],
```

Comment:
This should just be `angular()` without any casting

Reply:
`angular()` works, but the plugin doesn't work with a project, as I am getting the following error `angular is not a function` when testing, I am guessing this has to do with the output of the vite plugin, which produces an export in the form of `exports.default` which seems incompatible with Astro integration which is ESM based. any ideas?

Reply:
I am guessing this is the reason you had the workaround to begin with

Reply:
Yep, that was the reason. The vite plugin is currently shipped as commonjs. I think when I tried shipping ESM, it broke when used with Vite. Something more to investigate there, but I think we're good for now here.

Reply:
Let me add a comment to not this right now and then create an issue around this